tensorflow_addons/seq2seq/basic_decoder_test.py

```diff
@@ -82,8 +79,12 @@ def testStepWithTrainingHelperOutputLayer(self, use_output_layer):
                  tf.constant(0), first_inputs, first_state)
             batch_size_t = my_decoder.batch_size
 
-            self.assertTrue(isinstance(first_state, rnn_cell.LSTMStateTuple))
-            self.assertTrue(isinstance(step_state, rnn_cell.LSTMStateTuple))
+            self.assertTrue(
```

Comment:
This can probably be updated to use self.assertIsInstance()

Reply:
Thx.